script.js

```diff
@@ -13,3 +13,203 @@ navLinks.forEach((link) => {
+    languages: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    imgSrc: './Asset/oval.svg',
+    title: 'Tonic',
+    canopy: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    languages: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    imgSrc: './Asset/prof.svg',
+    title: 'Multi-Post Stories',
+    canopy: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    languages: ['HTML', 'CSS', 'JavaScript'],
+  },
+];
+
+const popupContainer = document.createElement('div');
+popupContainer.classList.add('popup-container');
+document.body.appendChild(popupContainer);
+
+popupContainer.addEventListener('click', (event) => {
+  if (event.target === popupContainer) {
+    popupContainer.style.display = 'none';
+  }
+});
+
+function popup(parram) {
+  if (parram < 0 || parram >= popy.length) {
+    // Handle the out-of-bounds case here
+    return null;
+  }
+  const html = `<div class="pone"> <span class="close">&times;</span><h1>${
+    popy[parram].title
+  }</h1><ul class="po"><li><span class="canop">${
+    popy[parram].canopy
+  }</span> <span class="role">${
+    popy[parram].role
+  }</span>  <span class="year"> ${
+    popy[parram].year
+  }</span> </li></ul><img src="${popy[parram].imgSrc}"></img><p>${
+    popy[parram].description
+  }</p> <ul class="hcj">
+  ${popy[parram].languages
+    .map((lang) => `<li class="pop_language">${lang}</li>`)
+    .join('')}
+</ul> <button class="popnewa">See live</button> <button class="popnewb">See source</button>
+</div> `;
+
+  return html;
+}
+const seeprobtns = document.querySelectorAll('.seepro');
+seeprobtns.forEach((btn, index) => {
+  btn.addEventListener('click', () => {
+    // Show the popup for the project at the corresponding index
+    const html = popup(index);
+    popupContainer.innerHTML = html;
+    popupContainer.style.display = 'block';
+
+    const closeBtn = document.querySelector('.close');
+    closeBtn.addEventListener('click', () => {
+      popupContainer.style.display = 'none';
+    });
+  });
+});
+const seeprobtn = document.querySelectorAll('.seepro');
+
+for (let i = 0; i < seeprobtn.length; i += 1) {
+  seeprobtn[i].addEventListener('click', () => {
+    popup(0);
+  });
+}
```

Comment:
As requested by the previous reviewer, kindly ensure that the popup window details appears when the user clicks (or taps) the button to check project details,

![image](https://user-images.githubusercontent.com/78251927/232156515-73a782da-d124-48e8-a76b-9c52c0648feb.png)

When I clicked on the button, the popup window did not appear. 
Project requirement
![image](https://user-images.githubusercontent.com/78251927/232156592-916dc9a0-a4eb-4ab6-8194-a4a72f76d444.png)

Part of the areas we can look into 👇 
- I can see you are calling event listeners on 

```JavaScript
const seeprobtns = document.querySelectorAll('.seepro');
```

However, I noticed that all the buttons have a different className which is not `seepro`